test/test_functions

```diff
@@ -836,7 +836,7 @@ create_filled_repo() {
 # @param  repo_dir   where to put the dummy dirents
 make_huge_repo() {
   local repo_dir=$1
-  $CVMFS_PYTHON2 ${TEST_ROOT}/common/mock_services/make_repo.py \
+  ${TEST_ROOT}/common/mock_services/make_repo.py                \
```

Comment:
you could also use `$CVMFS_SYS_PYTHON` as python selector

Reply:
Hmm, what's the advantage of `${CVMFS_SYST_PYTHON}` over `#!/usr/bin/env python`?